### DISCUSSION TITLE
Add NUT integration fails with a remote NUT server

### DISCUSSION THREAD

**Discussion by herwarth** [home-assistant/frontend#10019] [Configuration] [answered]
When adding the integration via the GUI I cannot set the alias.
I have NUT running on my QNAP NAS and QNAP defines a non configurable alias named qnapups.
So when running the following on a remote machine I get output:
```
upsc  qnapups@192.168.0.2
Init SSL without certificate database
battery.charge: 98
battery.charge.low: 10
battery.charge.warning: 50
battery.runtime: 3647
battery.runtime.low: 150
battery.type: PbAc
battery.voltage: 27.1
battery.voltage.nominal: 24.0
device.mfr: American Power Conversion
device.model: Smart-UPS 750
....
```
When looking in the code, the module accepts an alias to connect to.
So an input field to set the alias would be the solution. In this case the alias would be qnapups.

**Comment by spacegaier** [comment, marked as answer]
The GUI that is rendered to set up an integration via the GUI (they are called "config flows") is defined in the backend and the frontend just reacts to those backend definitions. So this topic would need to be addressed in the backend.

To me it sounds not really like a feature request, but more like a bug since the integration cannot be used in your scenario, so creating an issue in https://github.com/home-assistant/core sould be fine.
